package.json

```diff
@@ -39,7 +39,7 @@
   "scripts": {
     "build:docs": "jsdoc -c jsdoc.json lib/rewardops.js",
     "lint": "eslint --cache --ignore-path .gitignore .",
-    "test": "jest",
+    "test": "jest --watch",
```

Comment:
Can you remove this, please? It's used in automations and won't terminate with the `--watch` flag

Reply:
Ah crap, Dont know how that got in.